utils/correct_gps_outliers.py

```
@@ -112,31 +75,91 @@ def correct_gps_outliers(
 
     if not corrected_oids:
-        log_message("⚠️ No GPS outliers found or corrected.", messages, config=config)
-    else:
-        log_message(f"✅ Corrected {len(corrected_oids)} GPS outlier point(s).", messages, config=config)
+        logger.info("No GPS outliers found or corrected.", indent=1)
+        return
+
+    # Apply updates with error handling
+    failed_oids = set()
+    with cfg.get_progressor(total=len(corrected_oids), label="Correcting GPS outliers") as progressor:
+        with arcpy.da.UpdateCursor(oid_fc, fields) as cursor:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)